common/src/api/external/mod.rs

```diff
@@ -958,6 +958,14 @@ pub enum DiskState {
     Creating,
     /// Disk is ready but detached from any Instance
     Detached,
+    /// Disk is ready to receive blocks from an external source
+    ImportReady,
```

Comment:
Why is it important for `ImportReady` to exist over just having `Detached` be able to transition into one of the importing states?

Reply:
A few reasons:

1. `Detached` and `ImportReady` represent states from which only a subset of operations can be started. `ImportReady` is the state where the Disk is constantly attached to a Pantry and the operations that start from there all involve that Pantry. To transition from `Detached` straight to `ImportingFromBulkWrite` would mean attaching and detaching from a Pantry each time, which means a saga has to be invoked for that state change. It seemed lighter weight to park the disk in `ImportReady` instead exiting the disk_create saga, and to require the user to "finalize" the disk once all importing was done.
2. To "finalize" a disk means to transition out of `ImportReady` to `Detached`, and there's no way for that disk to again enter `ImportReady`. Once it's detached, it's ready for regular use with instances. Though, this supposes a world where a user will never again want to import blocks into a disk after they've done it the first time.